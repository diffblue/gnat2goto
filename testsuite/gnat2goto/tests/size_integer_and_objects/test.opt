ALL XFAIL gnat2goto fails when Size representation clause applied to standard types and their derivations or to objects.