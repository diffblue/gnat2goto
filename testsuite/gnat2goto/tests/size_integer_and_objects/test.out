+===========================GNAT BUG DETECTED==============================+
| GNU Ada (ada2goto) Assert_Failure tree_walk.adb:4673                     |
| Error detected at check_size.adb:1:11                                    |
| Please submit a bug report; see https://gcc.gnu.org/bugs/ .              |
| Use a subject line meaningful to you and us to track the bug.            |
| Include the entire contents of this bug box in the report.               |
| Include the exact command that you entered.                              |
| Also include sources listed below.                                       |
+==========================================================================+

Please include these source files with error report
Note that list may not be accurate in some cases,
so please double check that the problem can still
be reproduced with the set of files listed.
Consider also -gnatd.n switch (see debug.adb).

check_size.adb

compilation abandoned
